### ISSUE TITLE
Static arrays for possible_values doesn't work since 0.3.8

### ISSUE THREAD

**Issue by palfrey** [TeXitoi/structopt#354]
In structopt 0.3.7, the option `possible_values = &["always", "auto", "never"]` works (or at least compiles). In 0.3.8/0.3.9, it fails with the following error
```rust
error: expected `string literal` or `expression` after `=`
  --> src/bin/main.rs:10:25
   |
10 |         possible_values = &["always", "auto", "never"]
   |
```
Demo project is documented at https://gist.github.com/palfrey/9b7a44938b64550f53da6a3fe806ba2e

**Comment by CreepySkeleton** [comment]
Nice. `syn` doesn't recognize [array `[...]` expressions](https://docs.rs/syn/1.0.16/syn/struct.ExprArray.html) if the `full` feature is not enabled, but it's not mentioned in [`syn::Expr`](https://docs.rs/syn/1.0.16/syn/enum.Expr.html) documentation. We need `full` after all.

**Comment by CreepySkeleton** [comment]
The fix published in `0.3.11`
